### DISCUSSION TITLE
21.08.08.일 회의록

### DISCUSSION THREAD

**Discussion by damilog** [Team-IT-DA/Frontend#51] [회의록]
- [x] common 폴더 내 icons 폴터 images 폴더로 옮기는 것에 대해
- [x] product style 질문 아래 스타일 속성의 역할 궁금! (제니)
![image](https://user-images.githubusercontent.com/56783350/128620123-e7803a24-4e35-4c36-9daf-b82f6c0afc6c.png)
- [x] 원 단위 콤마 찍는 방법 공유
 `${productPrice.toLocaleString()}원`
- [x]  product card 테두리 할말? -> 이미지만 할 
- [x] side drawer 오른쪽에서 왼쪽으로 나오는 애니메이션 추가 가능한지?(디코) - 가능
- [x] 다음주차 업무 배분 및 미팅 일자 정하기

**Comment by ha3158987** [comment]
## Dico
- common 밑에 icons 폴더를 상위에 images로 옮기기
- 장바구니 애니메이션 추가하기
- 브랜드 스토리 페이지 글자 fix 시키기 (창 크기 줄여도 안 줄어들게)
- header에 링크 달기 
- 장바구니 삭제버튼에 로직 추가

**Reply by wjddnjswjd12** [reply]
Jenny
Card 전반적으로 리팩토링 필요
- Card Image에다가 border 먹이기
- webkit 부분 수정하기
- 장바구니쪽 card에는 description 빼기
- Review받은거 반영해보기
![image](https://user-images.githubusercontent.com/56783350/128630491-11d7f2bb-c30d-4c4f-8210-7e02665317ec.png)

**Comment by damilog** [comment]
product style 질문 아래 스타일 속성의 역할 궁금! (제니) 
```
white-space: nowrap; text-overflow: ellipsis; overflow: hidden; 
```
<img width="372" alt="스크린샷 2021-08-08 오후 8 20 27" src="https://user-images.githubusercontent.com/56783350/128630253-8faedfaa-0160-4a1d-95bd-f201d2cfbd50.png">
 상품의 타이틀이 너무 길 때 ..해주는 코드

**Comment by damilog** [comment]
## Daisy 
- position 오류 고치기
<img width="403" alt="스크린샷 2021-08-08 오후 8 21 11" src="https://user-images.githubusercontent.com/56783350/128630272-6d84ab2f-5dd7-4f97-9d58-45759690847d.png">

**Comment by damilog** [comment]
axios 설치 및 develop push 완료

**Comment by damilog** [comment]
- 프론트 다음 미팅 : 8월 11일 수요일 백엔드 미팅 끝나고 진행

- 업무 배분 
## Jenny 
- [x] 카드를 아름답게 리팩토링 해보겠습니다!
- [x] 데이지의 리뷰를 살펴보겠습니다! 
- [x] 리뷰창에 사진 클릭시 확대하는 모달(시간 남으면)

## Dico
- [ ] 헤더에 링크 붙이는거(라우팅)
- [ ] 장바구니 삭제 로직 구현
- [ ] 장바구니 수량 증가 로직 구현

## Daisy

- [ ] 장바구니 모달 추가(뷰 + 수량 증가) 
- [ ] 제품 클릭시 상세 페이지로 이동

**Comment by damilog** [comment]
https://github.com/Team-IT-DA/Frontend/wiki/8%EC%9B%94-1%EC%A3%BC%EC%B0%A8-%ED%9A%8C%EA%B3%A0(08.02-~-08.08)
